docs/data/base/components/input/input.md

```diff
@@ -0,0 +1,41 @@
+---
+product: base
+title: React Input unstyled component and hook
+components: InputUnstyled
+githubLabel: 'component: input'
+packageName: '@mui/base'
+---
+
+# Input
+
+<p class="description">Inputs let users enter and edit text.</p>
+
+## Basic usage
+
+```js
+import InputUnstyled from '@mui/base/InputUnstyled';
+```
+
+{{"demo": "UnstyledInputBasic.js", "defaultCodeOpen": false}}
+
+## Adornments
+
+Sometimes it is useful to add some kind of prefix, a suffix, or an action to an input.
+You can achieve this by using the adornment props.
+For instance, you can use an icon button to hide or reveal the password.
```

Comment:
```suggestion
For example, two very common use cases of adornments are when the input receives a specific unit of measure (like kilograms or currency) and when you want an icon button to hide or show a password.
```

To be a little bit more descriptive since we have the kg and $ examples in the demo contained as well.

Reply:
Much better 👍